houston/ardu/sandbox.py

```diff
@@ -360,22 +368,25 @@ def check_for_reached(m):
                         wp_to_traces[last_wp[0]] = trace
                         traces.append(trace)
 
-                        cm_directory = "command{}".format(last_wp[0])
-                        self.__copy_coverage_files(cm_directory)
-                        last_wp[0] = last_wp[1]
-                        wp_event.clear()
+                        # if appropriate, store coverage files
+                        if collect_coverage:
+                            cm_directory = "command{}".format(last_wp[0])
+                            self.__copy_coverage_files(cm_directory)
+                            last_wp[0] = last_wp[1]
+                            wp_event.clear()
```

Comment:
These last two lines should be outside of if

Reply:
Well spotted. Thanks!